### PR TITLE
Update Android target SDK version to 34, Bump iOS and Android app versions

### PR DESCRIPTION
**Pull request by kndoshn**
This is a part of the release process #129 and #130

### CODE REVIEW

**Comment by kndoshn** [comment]
The Google review got rejected because there was an empty action button.
I removed and resubmitted [ac50625](https://github.com/getditto/demoapp-chat/pull/131/commits/ac5062588a1c171dca25929d2a70d1b431b241c8)

This PR is ready for merge (again). 

IN_APP_EXPERIENCE-8781

**Review by bplattenburg — APPROVED** [review]
LGTM

File: Android/app/build.gradle [hunk 1]
```diff
@@ -8,13 +8,13 @@ plugins {
...
+    compileSdk 34
 
     defaultConfig {
         applicationId "live.dittolive.chat"
         minSdk 29
-        targetSdk 33
-        versionCode 27
+        targetSdk 34
+        versionCode 32
         versionName "1.0"
```
Comment (phatblat) [inline Android/app/build.gradle:18]: Should the Android `versionName` be set to `5.0.0` to match the version of the iOS app?

Comment (rajramsaroop) [reply Android/app/build.gradle:18]: I personally don't think it needs to match

Comment (phatblat) [reply Android/app/build.gradle:18]: No worries, just thought I'd ask since it looks like we haven't changed it from the default.

Comment (rajramsaroop) [inline Android/app/build.gradle:18]: I do think we should update the version though, 1.0.1 makes sense to me.

**Review by rajramsaroop — CHANGES_REQUESTED** [review]

File: Android/app/build.gradle (hunk 1 above)
Comment (rajramsaroop) [reply Android/app/build.gradle:18]: Updating the version does make sense though. I suggest 1.0.1

Comment (phatblat) [reply Android/app/build.gradle:18]:
```suggestion
        versionName "1.0.1"
```

**Pull request merged**